src/DynamicModel.cpp

```diff
@@ -87,6 +90,24 @@ base::Vector6d DynamicModel::calcCoriolisEffect(const base::Matrix6d &inertia_ma
     return -coriloisEffect;
 }
 
+base::Vector6d DynamicModel::caclDampingEffect(const UWVParameters& uwv_parameters, const base::Vector6d& velocity) const
```

Comment:
Just one minor detail, since the method also consider the Coriolis effect, what about to rename it in order to make it clear?
Something like `calcModelTypeEffect` or `calcDampingAndCoriolisEffect`?